Client/sdk/game/CModelInfo.h

```diff
@@ -28,6 +28,17 @@ class CBoundingBox
     float   fRadius;
 };
 
+enum class eModelInfoType : unsigned char
+{
+    ATOMIC = 1,
+    TIME = 3,
```

Comment:
Huh interesting, does gta skip over` = 2`?

Reply:
GTA SA don't use model info with enum 2

Reply:
Or this is special type for parts of destroyed car